### PR TITLE
add python file

### PR DESCRIPTION
**Pull request by AvinashDhillor**

### CODE REVIEW

**Comment by plskz** [comment]
rawr

**Review by plskz — CHANGES_REQUESTED** [review]
rename the file to `index.js`

File: javascrip.js [hunk 1]
```diff
@@ -1 +1,2 @@
-console.log("abcdef");
\ No newline at end of file
+console.log("abcdef");
```
Comment (plskz) [inline javascrip.js:1]: also you can delete this or rename abcdef to much better name

**Review by plskz — COMMENTED** [review]
test comment

**Comment by plskz** [comment]
test comment 3

**Review by plskz — APPROVED** [review]

**Comment by AvinashDhillor** [comment]
can you review python code

**Review by plskz — APPROVED** [review]

File: a.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+print("Hello World!")
+a = 10
+b = 10
```
Comment (plskz) [inline a.py:3]:
```suggestion
a = 20
b = 20
```

File: a.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+print(a + b);
```
Comment (plskz) [inline a.py]: much better
```suggestion
print("a + b is ", a + b);
```

**Review by plskz — CHANGES_REQUESTED** [review]

**Review by plskz — APPROVED** [review]

**Pull request merged**